### ISSUE TITLE
About the GPU-Operator update in Openshift

### ISSUE THREAD

**Issue by kousui-dev** [NVIDIA/gpu-operator#387]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

### 1. Quick Debug Checklist
- [No ] Are you running on an Ubuntu 18.04 node?
- [No(Openshift4.10) ] Are you running Kubernetes v1.13+?

### 1. Issue or feature description
Hello.
Currently, GPU-Operator1.8.2 is installed on Openshift4.10.

I would like to update GPU-Operatror to 1.10, but the status is not upgrade available when viewed from InstalledOperator on the OCP web console.
Even if I check from CLI, Installallplan is not created.

Is it possible to dynamically update from GPU-Operatror 1.8 to 1.10?
If it is not possible, should I uninstall GPU-Operatror1.8 and install 1.10?

thank you.

**Comment by shivamerla** [comment]
@kousui-dev If you use channel as `stable` in subscription, upgrade path to v1.10.x and v1.11.x should be available.

**Comment by kousui-dev** [comment]
@shivamerla 
thank you for answering.
But the GPU-Operator could not be updated.

I have checked the NVIDIA GPU Operator Channel for OCP4.10, but since the target of the Channel is from 1.9.0, I expect that the InstallPlan will not be created.
```
$ opm alpha list bundles registry.redhat.io/redhat/certified-operator-index:v4.10 gpu-operator-certified
WARN[0026] DEPRECATION NOTICE:
Sqlite-based catalogs and their related subcommands are deprecated. Support for
them will be removed in a future release. Please migrate your catalog workflows
to the new file-based catalog format. 
PACKAGE                 CHANNEL  BUNDLE                          REPLACES                        SKIPS  SKIP RANGE         IMAGE
gpu-operator-certified  stable   gpu-operator-certified.v1.10.0                                                            registry.connect.redhat.com/nvidia/gpu-operator-bundle@sha256:4d1268054559b7678028a7a66e36567c55417b729101ef131dec8d989cbef1d3
gpu-operator-certified  stable   gpu-operator-certified.v1.10.1  gpu-operator-certified.v1.10.0         >=v1.9.0 <v1.10.1  
...
```
Looking at the document below, it was stated that upgrading from v1.8 to v1.9 is not supported.
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/release-notes.html#id25

To organize the information, are there two ways to update from GPU-Operator 1.8.2 to 1.10?
(1) Uninstall 1.8.2, install 1.9, and update from 1.10
(2) Install 1.10 directly

thank you.

**Comment by shivamerla** [comment]
@kousui-dev yes, thanks for pointing out the namespace change issue. If you are fine with un-install, you can directly install v1.10 without an extra hop with v1.9.x. Also, v1.11.1 is available on OCP 4.10, that will be even better.
